.travis.yml

```diff
@@ -5,5 +5,7 @@ jdk: openjdk8
 env:
   SBT_OPTS="-Xmx4096M"
 script:
-  - sbt test
+  - export CDH_VERSION=5 sbt test
```

Comment:
Both CDH 5 and CDH 6 (solr 4 and solr7) tests are run in travis. These are big tests, and are a little flaky right now